### PR TITLE
Fix basePath in user.json & pet.json, add store.json

### LINKED ISSUE #75: Swagger Petstore example (spec 1.2) missing store.json
### Bug or feature request

- [x] Bug
- [ ] Feature request

### Description of feature (or steps to reproduce if bug)

The Swagger Petstore example (spec 1.2) seems to be broken: https://github.com/strongloop/loopback-swagger/tree/master/example/specs/1.2

It's missing `store.json` as referenced in `pet-store.json`: https://github.com/strongloop/loopback-swagger/blob/master/example/specs/1.2/pet-store.json#L14

### Link to sample repo to reproduce issue (if bug)

https://github.com/strongloop/loopback-swagger/tree/master/example/specs/1.2

### Expected result

`store.json` should not be missing

### Actual result (if bug)

`store.json` is not present in the folder: https://github.com/strongloop/loopback-swagger/tree/master/example/specs/1.2

### PR DESCRIPTION
**Pull request by wing328**
### Description

To fix #75 by adding store.json. Also fixed basePath in user.json, pet.json

#### Related issues

- connect to #75 

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by bajtos** [comment]
@slnode ok to test

**Comment by bajtos** [comment]
@wing328 thank you for the pull request, this is great!

Please sign our CLA, we cannot accept your contribution without that:

https://cla.strongloop.com/agreements/strongloop/loopback-swagger

**Comment by wing328** [comment]
> Please sign our CLA, we cannot accept your contribution without that:

Done

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by Amir-61** [comment]
@bajtos It seems this PR is ready to be landed. Could you please take a look... Thanks! 🙇

**Comment by Amir-61** [comment]
@wing328 You need to rebase the PR on top of master...

File: example/specs/1.2/store.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+          ]
+        },
+        "shipDate": {
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    }
+  }
+}
```
Comment (Amir-61) [inline example/specs/1.2/store.json]: Also please add one blank line at the end of the file.

File: example/specs/1.2/store.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    "application/json"
+  ],
+  "apis": [
+    {
+      "path": "/store/order/{orderId}",
+      "operations": [
+        {
+          "method": "GET",
+          "summary": "Find purchase order by ID",
+          "notes": "For valid response try integer IDs with value <= 5. Anything above 5 or nonintegers will generate API errors",
```
Comment (Amir-61) [inline example/specs/1.2/store.json]: Nitpick: Can we change all `nonintegers ` to `non-integers` please?

File: example/specs/1.2/store.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            {
+              "code": 404,
+              "message": "Order not found"
+            }
+          ]
+        },
+        {
+          "method": "DELETE",
+          "summary": "Delete purchase order by ID",
+          "notes": "For valid response try integer IDs with value < 1000.  Anything above 1000 or nonintegers will generate API errors",
```
Comment (Amir-61) [inline example/specs/1.2/store.json]: ditto

**Comment by Amir-61** [comment]
@wing328 you may address these minor comments and re-base it on top of master; then I will land it, since it seems @bajtos has already approved the patch and he is out of office today.

**Comment by bajtos** [comment]
Thank you @Amir-61 for chiming in. I have addressed your comments myself. Let's wait for CI results before landing.

**Comment by bajtos** [comment]
Landed, thank you for the contribution!

**Pull request merged**